libs/pages/application/src/lib/feature/page-variables-feature/page-variables-feature.tsx

```diff
@@ -30,7 +30,9 @@ export function PageVariablesFeature() {
   useDocumentTitle('Environment Variables – Qovery')
   const dispatch = useDispatch<AppDispatch>()
   const { applicationId = '' } = useParams()
-  const [placeholder] = useState(environmentVariableFactoryMock(5))
+  const [placeholder] = useState<EnvironmentVariableSecretOrPublic[]>(
```

Comment:
Seems to be a simple variable if you don't use the `setState` no?

Reply:
I use this placeholder in a useEffect below and as it is generated by a factoryMock, the value of placeholder would change at every render and then retrigger the useEffect in loop...

Reply:
useMemo can be a solution to avoid the render ?